frontend/src/v5/ui/components/shared/dashedContainer/dashedContainer.component.tsx

```diff
@@ -14,11 +14,20 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+import { Container } from './dashedContainer.styles';
 
-import styled from 'styled-components';
+type DashedContainerProps = {
+	children: any;
+	className?: string;
+	borderRadius?: number;
+	strokeColor?: string;
+	strokeWidth?: string;
+	dashSize?: number;
+	gapSize?: number;
+};
 
-export const HeaderButtonsGroup = styled.div`
-	& > :last-child {
-		margin-right: 0;
-	}
-`;
+export const DashedContainer = ({ children, ...dashedOptions }: DashedContainerProps) => (
```

Comment:
I would move DashedContainer to controls. Its pretty general.

Reply:
Done and I also create the storybook component